File: transaction/python/TransactionIdInfo.py
import requests
import json

# set the url
url = "https://api.deversifi.com/v1/pub/transaction/{tx_hash}"

# set the parameters
url = url.format(
    tx_hash="0x1686cd265bf021a38f9a70531016e28cc8dabe41a6f9639eda5b2c3873cdb9d5")

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers).text)

# print the response from dictionary
print(json.dumps(response, indent=2))

File: transaction/js/TransactionIdInfo.js
var request = require("request");
// set parameters
const txHash = "0x450a8e0db4966a9b05bcbba038a4b1d74c30b0b178ff76f325dc7da9c56f305a";
// set url
const url = "https://api.deversifi.com/v1/pub/transaction/" + txHash;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});
